inductiva/resources/machine_groups.py

```diff
@@ -1,16 +1,16 @@
 """Functions to manage or retrieve user resources."""
+from typing import Optional
+from absl import logging
 import inductiva
 import inductiva.client
 from inductiva.client.apis.tags import compute_api
 from inductiva.utils import format_utils
 from inductiva import resources
 
 
-def estimate_machine_cost(machine_type: str,
-                          zone: str = "europe-west1-b",
```

Comment:
To be honest, the removal of the zone parameter seems cumbersome, but since we have already discuss it, I think we should open an issue to not forget and maybe tag this PR and the one on the backend so that we know where we should add it in.

Reply:
Yep, I just did it [here](https://github.com/inductiva/inductiva-web-api/issues/488).